components/script/dom/node.rs

```diff
@@ -2490,6 +2490,11 @@ pub enum ChildrenMutation<'a> {
         next: Option<&'a Node>,
     },
     ReplaceAll { removed: &'a [&'a Node], added: &'a [&'a Node] },
+    /// Mutation for when a Text node's data is modified.
+    /// This doesn't change the structure of the list, which is what the other
+    /// variants' fields are stored for at the moment, so this can just have no
+    /// arguments.
```

Comment:
"this can have no fields" is more consistent, thanks for catching that! hm, "Text" looks nicer but the other ones are describing actions and "Text" is just a noun. maybe "ChangeText" (the others are present tense infinitives)? then it fits with us calling content_changed and with this applying for Text nodes, maybe